src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_ObsoleteAttribute.vb

```diff
@@ -1490,5 +1491,110 @@ BC30911: 'Get' accessor of 'Public Property ExceptionalProp As String' is obsole
+            Dim comp0 = CreateCompilationWithMscorlib(ExperimentalAttributeSource)
+            comp0.AssertNoDiagnostics()
+            Dim ref0 = comp0.EmitToImageReference()
+
+            Dim source1 =
+<compilation>
+    <file><![CDATA[
+Imports Windows.Foundation.Metadata
+Namespace N
+    Public Class A
+        <Experimental>
+        Public Sub F(ParamArray args As Object())
+        End Sub
+        <Experimental>
+        Public Class B
+            Inherits A
+        End Class
+        <Experimental>
+        Private ReadOnly Property P As Object
+            Get
+                Return Nothing
+            End Get
+        End Property
+        Private Property Q As Object
+            Get
+                Return Nothing
+            End Get
+            <Experimental>
+            Set
+            End Set
+        End Property
+        Private Sub M()
+            F()
+            Q = P
+        End Sub
+    End Class
+End Namespace
+]]>
+    </file>
+</compilation>
+            Dim comp1 = CreateCompilationWithMscorlib(source1, references:={ref0})
+            comp1.AssertTheseDiagnostics(<errors>
+BC42380: 'Public Sub F(ParamArray args As Object())' is for evaluation purposes only and is subject to change or removal in future updates.
+            F()
+            ~~~
+BC42380: 'Private Property Set Q(Value As Object)' is for evaluation purposes only and is subject to change or removal in future updates.
+            Q = P
+            ~~~~~
+BC42380: 'Private ReadOnly Property P As Object' is for evaluation purposes only and is subject to change or removal in future updates.
+            Q = P
+                ~
+     </errors>)
+
+            Dim source2 =
+<compilation>
+    <file><![CDATA[
+Imports N
+Class C
+    Shared Sub Main()
+        Dim a As A = New A.B()
+        a.F()
+    End Sub
+End Class
+]]>
+    </file>
+</compilation>
+            Dim comp2A = CreateCompilationWithMscorlib(source2, references:={ref0, comp1.EmitToImageReference()})
+            comp2A.AssertTheseDiagnostics(<errors>
+BC42380: 'N.A.B' is for evaluation purposes only and is subject to change or removal in future updates.
+        Dim a As A = New A.B()
+                         ~~~
+BC42380: 'Public Sub F(ParamArray args As Object())' is for evaluation purposes only and is subject to change or removal in future updates.
+        a.F()
+        ~~~~~
+     </errors>)
+
+            Dim comp2B = CreateCompilationWithMscorlib(source2, references:={ref0, New VisualBasicCompilationReference(comp1)})
+            comp2B.AssertTheseDiagnostics(<errors>
+BC42380: 'N.A.B' is for evaluation purposes only and is subject to change or removal in future updates.
+        Dim a As A = New A.B()
+                         ~~~
+BC42380: 'Public Sub F(ParamArray args As Object())' is for evaluation purposes only and is subject to change or removal in future updates.
+        a.F()
+        ~~~~~
+     </errors>)
+        End Sub
```

Comment:
Seems like fewer tests in VB. In particular, no `TestDeprecatedAndExperimentalAndObsoleteAttributes`. Is that intentional?